app/celer-g4/RunInput.cc

```diff
@@ -131,6 +131,7 @@ inp::Problem load_problem(RunInput const& ri)
         d.export_files.physics = ri.physics_output_file;
         d.export_files.offload = ri.offload_output_file;
         d.timers.action = ri.action_times;
+        d.perfetto_file = ri.tracing_file;
```

Comment:
This still isn't used in the `setup/Problem.cc`... see my comment on the PR. We should add a to-do there at least:
```
diff --git a/src/celeritas/setup/Problem.cc b/src/celeritas/setup/Problem.cc
index 0704e1924..fc53eab7e 100644
--- a/src/celeritas/setup/Problem.cc
+++ b/src/celeritas/setup/Problem.cc
@@ -468,6 +468,8 @@ ProblemLoaded problem(inp::Problem const& p, ImportData con>

     result.output_file = p.diagnostics.output_file;

+    // TODO: timers, counters, perfetto_file
+
     if (p.diagnostics.action)
     {
         ActionDiagnostic::make_and_insert(*core_params);
```

Reply:
Yep, I'll take a look at moving the instantiation of the session inside Celeritas

Reply:
@esseivaju did you mean to say as part of this PR? I think for this one, just adding this note (and merging develop) is good.

Reply:
Sure let's follow up in a later PR